changelog.txt

```diff
@@ -1,123 +1,314 @@
-0.4.1
-
+=========
+Changelog
+=========
+
+The **garnett** package follows `semantic versioning <https://semver.org/>`_.
+
+Version 0.5
+===========
+
+Highlights
+----------
+	- Remove acceleration as supported property.
+	- Remove the read_gsd_shape_data flag from GSD reader.
+	- Raise AttributeError if accessing an invalid property.
+	- Add GETAR file reader/writer.
+	- Add support for various particle properties
+	- Extend reader/writer shape support
+	- Add automatic format detection to reader/writer
+	
+
+[0.5.0] -- 2019-0x-xx
+---------------------
+Added
++++++
+
+	- Add GETAR file reader/writer.
+	- Rowan is now a dependency.
+	- Add shape_dict representation to Shape classes.
+	- Add support for particle properties: 
+		- mass 
+		- charge 
+		- diameter
+		- image
+		- moment of inertia 
+		- angular momentum
+	- Add support for reading/writing shapes in GSD via hpmc state.
+	- Add universal reader/writer with format detection.
+	- Extend list of supported shape classes: 
+		- Ellipsoid 
+		- orientable spheres
+		- polygons
+		- spheropolygon
```

Comment:
Be consistent with singular or plural form (ellipsoid vs. ellipsoids, polyhedron vs. polyhedra).

Reply:
Addressed in 8551feb